### DISCUSSION TITLE
Rewards

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2067] [Rewards]
social media loyal follower ibxc and points rewards;

1) Twitter
2) discord
3) telegram

awards;
1) Twitter loyal follower reward:
5000 points and 1000 ibxc

2)discord loyal follower reward:
5000 and 1000 ibxc

3) telegram loyal follower reward:
5000 and 1000 ibxc

**Comment by yolandadadada** [comment]
We appreciate your suggestions for us and look forward to the second round of airdrops.
